server.js

```diff
@@ -124,7 +126,7 @@ function render (req, res) {
   })
 }
 
-app.get('*', isProd ? render : (req, res) => {
+app.get('*', (req, res) => {
   readyPromise.then(() => render(req, res))
```

Comment:
That's exactly why that ternary was there, to avoid the extra overhead of an unneeded promise.